maquetado/scss/tech.scss

```diff
@@ -24,6 +24,18 @@ $tech-logo-size: 100px;
   }
 }
 
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.wrap {
+  display: flex;
+  justify-content: space-between;
+  width: 60%;
```

Comment:
We don't use percentages for width. In Wolox's tech guides we have that if the width is greater than 250px then we have to set it like so: 

```
width: 100%;
max-width: some size in px;
```